package.json

```diff
@@ -37,12 +37,16 @@
     "eslint-plugin-import": "2.17.2",
     "eslint-plugin-prettier": "3.0.1",
     "eslint-plugin-react": "7.13.0",
-    "husky": "2.3.0",
+    "husky": "^2.3.0",
     "jest": "24.8.0",
     "lerna": "3.13.4",
     "lint-staged": "8.1.4",
     "prettier": "1.16.4",
-    "stylelint": "^9.10.1"
+    "stylelint": "^9.10.1",
+    "tslint": "^5.11.0",
```

Comment:
why don't we use eslint for this?

https://github.com/palantir/tslint/issues/4534

we use eslint to lint all js and ts files in grant

Reply:
nice thanks for the tip, i removed tslint